src/SFA.DAS.ApplyService.Web/Infrastructure/Services/PagesWithSectionsFlowService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.ApplyService.Domain.Apply;
+using SFA.DAS.ApplyService.Domain.Entities;
+using SFA.DAS.ApplyService.Web.Infrastructure.Interfaces;
+using SFA.DAS.ApplyService.Web.ViewModels.Roatp;
+
+namespace SFA.DAS.ApplyService.Web.Infrastructure.Services
+{
+    public class PagesWithSectionsFlowService: IPagesWithSectionsFlowService
```

Comment:
As discussed, this is a good candidate for some unit test coverage in the tech debt story